src/odin/main.py

```diff
@@ -43,6 +44,10 @@ def main(argv=None):
     config.define('enable_cors', default=False,
                   option_help='Enable cross-origin resource sharing (CORS)')
     config.define('cors_origin', default='*', option_help='Specify allowed CORS origin')
+    config.define('log_server', default=None, option_help="Graylog server address and :port")
```

Comment:
Suggest these config options become `graylog_server`, `graylog_server_level` and `graylog_static_fields` to isolate them from future logging options?

Reply:
Updated. I went with `graylog_logging_level`, though.